_layouts/download_bak.html

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
is this used? _bak suffix makes me think maybe its not?

Reply:
don't think so. This is def a roy artifact though. @roybotbot do we need this?

Reply:
i removed it. seemed to be a legacy artifact. we can restore from the previous commit if needed.